### DISCUSSION TITLE
Clustering polygons

### DISCUSSION THREAD

**Discussion by aluhamaa** [opengeos/leafmap#902] [Q&A] [answered]
Hi
Is it somehow possible to cluster polygons in the similar way to clustering points, like in the 50_marker_cluster example? I have a global dataset of polygon sizes up to few hundred km's and all plotting tools start to struggle when plotting more than 200k polygons, but it would be great to support at least 2million.

**Comment by giswqs** [comment, marked as answer]
ipyleaflet is not  desgined for supporting large vector datasets. The recommended way to convert vector dataset to PMTIles. See https://leafmap.org/notebooks/82_pmtiles/

https://maps.gishub.org/maplibre/overture_1.html

**Reply by aluhamaa** [reply]
Perfect, thank you very much!
